archlinuxcn/lceda-pro/lilac.yaml

```
@@ -0,0 +1,18 @@
+maintainers:
+  - github: taotieren
+
+build_prefix: extra-x86_64
+
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver)
+
+post_build_script: |
+  git_pkgbuild_commit()
+  update_aur_repo()
+
+update_on:
+  - source: htmlparser
+    url: "https://lceda.cn/page/download?src=index"
+    xpath: "//table/tbody/tr[3]/td[2]/div/span[1]"
```

```suggestion
    xpath: "//table/tr[3]/td[2]/div/span[1]"
```
不好意思我刚才修改了一下建议，你可能没有看到，我发现HTML里没有`tbody`，这个应该是浏览器自动添加的，使用`htmlparser`的话我觉得应当删去。